src/formio.component.ts

```diff
@@ -161,6 +167,14 @@ export class FormioComponent implements OnInit {
             }
         });
     }
+    onPrevPage(data: any) {
+        this.alerts.setAlerts([]);
```

Comment:
Nevermind on that comment. I didn't notice this was on the angular 2 wrapper. On my phone and it is hard ton tell.